darkmoon/api/v1/metadata/Exception_Response.py

```diff
@@ -0,0 +1,22 @@
+class MissingHashException(Exception):
```

Comment:
I want created Exceptions for inherit the HTTPException model so that we still have the status_code and detail fields.